src/core_modules/capture-core/components/TopBarActions/TopBarActions.container.js

```diff
@@ -16,9 +16,9 @@ const defaultContext = {
 };
 
 const defaultDialogProps = {
-    header: i18n.t('Unsaved changes'),
-    text: i18n.t('Leaving this page will discard the changes you made to this event.'),
-    confirmText: i18n.t('Yes, discard'),
+    header: i18n.t('Discard unsaved changes?'),
```

Comment:
The same suggestion here about reusing `header`, `text`, `destructiveText` and `cancelText` const values.